test/knot-data/zones/test.zone

```diff
@@ -0,0 +1,19 @@
+; SPDX-License-Identifier: LGPL-2.1-or-later
+$TTL 86400
+$ORIGIN test.
+
+@ IN SOA ns1.signed.test. root.signed.test. (
+    42   ; serial
+    3H   ; refresh
+    15M  ; retry
+    1W   ; expire
+    1D   ; minimum TTL
+)
+
+; NS info
+@                     NS ns1.signed.test.
```

Comment:
just ``@   NS  ns1.signed`` could be used as relative reference. When ns1.signed is relative, maybe nameservers could be relative as well.